src/main/java/com/ea/restaurant/entities/BaseEntity.java

```diff
@@ -9,12 +14,15 @@
 @EqualsAndHashCode(of = "id", callSuper = false)
 @Getter
 @Setter
+@MappedSuperclass
 public abstract class BaseEntity {
-
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
   private long id;
-  private long updatedBy;
-  private long createdBy;
-  private Status entityStatus;
+
+  @Nonnull private long updatedBy;
```

Comment:
Use objects as first option instead of premitive values, since they have more features and you don't have huge application that be affected by performance for now. So go ahead and use `Long` implementation from java instead

Reply:
Done.